collections/_drafts/agent-forwarding.md

```diff
@@ -140,8 +141,7 @@ keyring into the remote container gets done automatically by the extension.
 
 \[\[endnote\]\]
 
-\`\`\`
-SCP
+\`\`\` SCP
```

Comment:
admittedly, still very difficult to know i was going for here, but relevant:

```suggestion
<dfn id="scp"><abbr title="Secure Copy Protocol">SCP</abbr> </dfn> is a
  means of securely transferring [computer files](https://en.wikipedia.org/wiki/Computer_file) between a local [host](https://en.wikipedia.org/wiki/Server_(computing)) and a remote host or between two remote hosts. It is based on the [Secure Shell](https://en.wikipedia.org/wiki/Secure_Shell) (SSH) protocol.[[1]](https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-1) "SCP" commonly refers to both the Secure Copy Protocol and the program itself.[[2]](https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-Pechanec-2)
```

```suggestion
[SCP]{#scp .dfn} is a means of securely transferring \[computer
files\](https://en.wikipedia.org/wiki/Computer_file) between a local
\[host\](https://en.wikipedia.org/wiki/Server\_(computing)) and a remote
host or between two remote hosts. It is based on the \[Secure
Shell\](https://en.wikipedia.org/wiki/Secure_Shell) (SSH)
protocol.\[\[1\]\](https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-1)
\"SCP\" commonly refers to both the Secure Copy Protocol and the program
itself.\[\[2\]\](https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-Pechanec-2)
```


we should pretend that we already support this dfn syntax:

🔗 https://pandoc.org/try/?params=%7B%22text%22%3A%22%3Cdfn+id%3D%5C%22scp%5C%22%3E%3Cabbr+title%3D%5C%22Secure+Copy+Protocol%5C%22%3ESCP%3C%2Fabbr%3E%3C%2Fdfn%3E+is+a%5Cn++means+of+securely+transferring+%5Bcomputer+files%5D%28https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FComputer_file%29+between+a+local+%5Bhost%5D%28https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FServer_%28computing%29%29+and+a+remote+host+or+between+two+remote+hosts.+It+is+based+on+the+%5BSecure+Shell%5D%28https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FSecure_Shell%29+%28SSH%29+protocol.%5B%5B1%5D%5D%28https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FSecure_copy_protocol%23cite_note-1%29+%5C%22SCP%5C%22+commonly+refers+to+both+the+Secure+Copy+Protocol+and+the+program+itself.%5B%5B2%5D%5D%28https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FSecure_copy_protocol%23cite_note-Pechanec-2%29%22%2C%22to%22%3A%22commonmark_x%22%2C%22from%22%3A%22html%22%2C%22standalone%22%3Atrue%2C%22embed-resources%22%3Afalse%2C%22table-of-contents%22%3Afalse%2C%22number-sections%22%3Afalse%2C%22citeproc%22%3Afalse%2C%22html-math-method%22%3A%22plain%22%2C%22wrap%22%3A%22auto%22%2C%22highlight-style%22%3Anull%2C%22files%22%3A%7B%7D%2C%22template%22%3Anull%7D

Reply:
```suggestion
:::excerpt{.quote}

<abbr title="Secure Copy Protocol">[SCP]{#scp .dfn}</abbr> is a means of
securely transferring [computer files][] between a local [host][] and a remote
host or between two remote hosts. It is based on the [Secure Shell][]  (SSH)
protocol.[^1] "SCP" commonly refers to both the Secure Copy Protocol and the
program itself.[^2]

:::

[computer files]: https://en.wikipedia.org/wiki/Computer_file
[host]: https://en.wikipedia.org/wiki/Server\_(computing))
[Secure Shell]: https://en.wikipedia.org/wiki/Secure_Shell)

[^1]: https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-1
[^2]: https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-Pechanec-2)
```